lib/src/configuration_interfaces/countly_config_apm.dart

```diff
@@ -0,0 +1,43 @@
+/// This class holds APM specific configurations to be used with CountlyConfig class and serves as an interface.
+/// You can chain multiple configurations.
+class CountlyConfigApm {
+  /// private variables.
+  bool _enableForegroundBackground = false;
+  bool _recordAppStartTime = false;
+  bool _enableManualAppLoaded = false;
+  int _startTSOverride = 0;
+
+  /// getters
+  bool get enableForegroundBackground => _enableForegroundBackground;
+  bool get recordAppStartTime => _recordAppStartTime;
+  bool get enableManualAppLoaded => _enableManualAppLoaded;
+  int get startTSOverride => _startTSOverride;
+
+  /// setters / methods
+
+  /// Enables the automatic tracking of app foreground and background durations.
+  CountlyConfigApm enableForegroundBackgroundTracking() {
+    _enableForegroundBackground = true;
+    return this;
+  }
+
+  /// Enables the automatic tracking of app start time.
+  CountlyConfigApm enableAppStartTimeTracking() {
+    _recordAppStartTime = true;
```

Comment:
to comply with the function name, maybe _trackAppStartTime but just an idea

Reply:
i think so too. keeping the old one but changing looks more appealing indeed